app/config/keymaps.js

```diff
@@ -0,0 +1,57 @@
+const {readFileSync} = require('fs');
+const {darwinKeys, win32Keys, linuxKeys} = require('./paths');
+
+const commands = {};
+const keys = {};
+
+const _setKeysForCommands = function (keymap) {
+  for (const command in keymap) {
+    if (command) {
+      commands[command] = keymap[command].toLowerCase();
+    }
+  }
+};
+
+const _setCommandsForKeys = function (commands) {
+  for (const command in commands) {
+    if (command) {
+      keys[commands[command]] = command;
+    }
+  }
+};
+
+const _import = function (customsKeys) {
+  const path = () => {
```

Comment:
Why not move this to `app/config/paths.js` and just expose it as `defaultPlatformKeyPath`?

Is there a need to get other platforms' keys?